src/Components/Patient/SampleTest.tsx

```diff
@@ -25,6 +30,7 @@ const initForm: SampleTestModel = {
   diagnosis: "",
   diff_diagnosis: "",
   doctor_name: "",
+  testing_facility: "",
```

Comment:
Codacy found an issue: [Identifier 'testing_facility' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5828652)